src/mojs.babel.js

```diff
@@ -0,0 +1,51 @@
+import h from './h';
+import shapesMap from 'shapes/shapesMap';
+import stagger from './stagger';
+import tweener from 'tween/tweener';
+import easing from 'easing/easing';
+import Shape from './shape';
+import ShapeSwirl from './shape-swirl';
+import Burst from './burst';
+import Html from './html';
+import Spriter from './spriter';
+import MotionPath from './motion-path';
+import Tween from 'tween/tween';
+import Timeline from 'tween/timeline';
+import Tweenable from 'tween/tweenable';
+import Thenable from './thenable';
+import Tunable from './tunable';
+import Delta from 'delta/delta';
+import Deltas from 'delta/deltas';
+import Module from './module';
+
+export default {
+  revision: build.revision,
+  isDebug: true,
```

Comment:
@xavierfoucrier is this `isDebug` really suppose to be in here? I see that we to a check in h.coffe to see if isDebug is set or not. Not sure how this would be used from the outside though.

If I import mojs from @mojs/core, and then run mojs.isDebug = false, I guess that will result in the logs, warnings and errors will not be visible.

So maybe is useful in some situations..

Reply:
Yup, I will have a look at it, good catch :wink:

Reply:
Feels like this should be set with node env settings instead. If node settings is set to production, there is no logs, else we provide logs to the developer. Same as other frameworks like react handles it. Or rather provide a better debug experience in dev mode. But that's maybe a bit of a bigger scope then this bool.